### ISSUE TITLE
Example does not work on Windows

### ISSUE THREAD

**Issue by Baukebrenninkmeijer** [sdv-dev/RDT#114]
* Reversible Data Transforms version: 0.2.3
* Python version: 3.7.6
* Operating System: Windows

### Description

When installing clean, it uses pandas 0.25.3. However, this does not seem to support the implicit casting of datetypes to floats and returns the error below.

EDIT: Newer versions also don't seem to support it (1.0.5)

### Commands
```
import rdt
demo = rdt.get_demo()
from rdt.transformers import DatetimeTransformer
transformer = DatetimeTransformer()
transformer.fit(demo['3_datetime'])
``` 
which returns this error
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-5-3f464036bdba> in <module>
----> 1 transformer.fit(demo['3_datetime'])

~\AppData\Local\Continuum\anaconda3\lib\site-packages\rdt\transformers\datetime.py in fit(self, data)
     53             data = pd.Series(data)
     54 
---> 55         transformed = self._transform(data)
     56 
     57         if self.nan == 'mean':

~\AppData\Local\Continuum\anaconda3\lib\site-packages\rdt\transformers\datetime.py in _transform(datetimes)
     38         """Transform datetime values to integer."""
     39         nulls = datetimes.isnull()
---> 40         integers = datetimes.astype(int).astype(float).values
     41         integers[nulls] = np.nan
     42 

~\AppData\Local\Continuum\anaconda3\lib\site-packages\pandas\core\generic.py in astype(self, dtype, copy, errors, **kwargs)
   5880             # else, only a single dtype is given
   5881             new_data = self._data.astype(
-> 5882                 dtype=dtype, copy=copy, errors=errors, **kwargs
   5883             )
   5884             return self._constructor(new_data).__finalize__(self)

~\AppData\Local\Continuum\anaconda3\lib\site-packages\pandas\core\internals\managers.py in astype(self, dtype, **kwargs)
    579 
    580     def astype(self, dtype, **kwargs):
--> 581         return self.apply("astype", dtype=dtype, **kwargs)
    582 
    583     def convert(self, **kwargs):

~\AppData\Local\Continuum\anaconda3\lib\site-packages\pandas\core\internals\managers.py in apply(self, f, axes, filter, do_integrity_check, consolidate, **kwargs)
    436                     kwargs[k] = obj.reindex(b_items, axis=axis, copy=align_copy)
    437 
--> 438             applied = getattr(b, f)(**kwargs)
    439             result_blocks = _extend_blocks(applied, result_blocks)
    440 

~\AppData\Local\Continuum\anaconda3\lib\site-packages\pandas\core\internals\blocks.py in astype(self, dtype, copy, errors, values, **kwargs)
    557 
    558     def astype(self, dtype, copy=False, errors="raise", values=None, **kwargs):
--> 559         return self._astype(dtype, copy=copy, errors=errors, values=values, **kwargs)
    560 
    561     def _astype(self, dtype, copy=False, errors="raise", values=None, **kwargs):

~\AppData\Local\Continuum\anaconda3\lib\site-packages\pandas\core\internals\blocks.py in _astype(self, dtype, **kwargs)
   2264 
   2265         # delegate
-> 2266         return super()._astype(dtype=dtype, **kwargs)
   2267 
   2268     def _can_hold_element(self, element):

~\AppData\Local\Continuum\anaconda3\lib\site-packages\pandas\core\internals\blocks.py in _astype(self, dtype, copy, errors, values, **kwargs)
    641                     # _astype_nansafe works fine with 1-d only
    642                     vals1d = values.ravel()
--> 643                     values = astype_nansafe(vals1d, dtype, copy=True, **kwargs)
    644 
    645                 # TODO(extension)

~\AppData\Local\Continuum\anaconda3\lib\site-packages\pandas\core\dtypes\cast.py in astype_nansafe(arr, dtype, copy, skipna)
    668         raise TypeError(
    669             "cannot astype a datetimelike from [{from_dtype}] "
--> 670             "to [{to_dtype}]".format(from_dtype=arr.dtype, to_dtype=dtype)
    671         )
    672 

TypeError: cannot astype a datetimelike from [datetime64[ns]] to [int32]
```

**Comment by csala** [comment]
Thanks for reporting this @Baukebrenninkmeijer 

I'm not able to reproduce the mentioned behavior: both the stable and development versions of `rdt` install `pandas==0.25.3`, as you mentioned, but neither of them seem to produce any error on the demo data on my setup.

Can you post the output of `pip freeze` to see all the exact versions that you are using on the rest of the libraries?

Also, it seems like you are running on Windows, which we do not actively test against and do not formally support, so that might be the source of the problem. Would you have the chance to try to reproduce the error on MacOS or Linux?

**Comment by Baukebrenninkmeijer** [comment]
I now created a new environment to verify this. The same happened unfortunately. I changed to title of the issue to reflect what is actually the problem, which wasn't Pandas version. 

My pip freeze currently looks like this:
```
attrs==19.3.0
backcall==0.2.0
bleach==3.1.5
certifi==2020.6.20
colorama==0.4.3
copulas==0.3.1
cycler==0.10.0
decorator==4.4.2
defusedxml==0.6.0
docutils==0.14
entrypoints==0.3
exrex==0.10.5
Faker==1.0.8
graphviz==0.14.1
importlib-metadata==1.7.0
ipykernel==5.3.4
ipython==7.16.1
ipython-genutils==0.2.0
ipywidgets==7.5.1
jedi==0.17.2
Jinja2==2.11.2
joblib==0.16.0
jsonschema==3.2.0
jupyter-client==6.1.6
jupyter-core==4.6.3
kiwisolver==1.2.0
MarkupSafe==1.1.1
matplotlib==3.2.1
mistune==0.8.4
nbconvert==5.6.1
nbformat==5.0.7
notebook==6.0.3
numpy==1.19.1
packaging==20.4
pandas==0.24.2
pandocfilters==1.4.2
parso==0.7.0
pickleshare==0.7.5
prometheus-client==0.8.0
prompt-toolkit==3.0.5
Pygments==2.6.1
pyparsing==2.4.7
pyrsistent==0.16.0
python-dateutil==2.8.1
pytz==2020.1
pywin32==228
pywinpty==0.5.7
pyzmq==19.0.1
rdt==0.2.3
scikit-learn==0.22.2.post1
scipy==1.2.3
sdmetrics==0.0.2.dev0
sdv==0.3.6
seaborn==0.10.1
Send2Trash==1.5.0
six==1.15.0
terminado==0.8.3
testpath==0.4.4
text-unidecode==1.2
tornado==6.0.4
traitlets==4.3.3
wcwidth==0.2.5
webencodings==0.5.1
widgetsnbextension==3.5.1
wincertstore==0.2
zipp==3.1.0
```

When running on WSL, it does seem to work. So might be something in the pandas windows implementation?

**Comment by csala** [comment]
This was fixed on PR #140
